c7n/resources/ebs.py

```diff
@@ -514,6 +514,82 @@ def process_resource_set(self, resource_set):
                 "Cross region copy complete %s", ",".join(copy_ids))
 
 
+@Snapshot.action_registry.register('remove-create-volume-permissions')
```

Comment:
rename `set-permissions` with explicit option/param for remove and add permissions

Reply:
So I modified this to be a more generic 'set-create-volume-permissions' action, allowing us to both add and remove permissions, which I hope is in line with what you were suggesting. Two things I'm not entirely sure about:

- The interactions with the 'cross-account' filter is a bit over my head. Please sanity check, and tell me if it makes sense as it is.

- The 'remove_accounts' and 'add_accounts' parameters may be lists, as per the updated schema. My understanding is that this is unlike the 'ami:remove-launch-permissions' action's 'accounts' parameter, which in its schema is declared to be either 'matched' or a single account id string?

Reply:
Set-permissions as name there are no other relevant permissions then create which makes it redundant

Reply:
Renamed it. We'll talk again when they introduce a second permission on that object... just kidding. ;-)

Did the rest of this look good to you?

Reply:
its only been stable for 12 years, i wouldn't hold your breath ;-) other consideration is add/remove sans account for consistency with other set perm style actions.

Reply:
Care to elaborate what you mean by sans account in this context? Let me know what other set perm action I should look at for a reference.

Reply:
Thanks for elaborating on this offline. I've added a commit that renames 'remove_accounts' to 'remove' and 'add_accounts' to 'add'. Generally a fan of brevity as well. :-)